### PR TITLE
Fix new lint errors

### PR DESCRIPTION
**Pull request by thornbill**
**Changes**
Fixes the new eslint errors introduced by the upgrade in #3405

**Issues**
N/A

### CODE REVIEW

**Comment by thornbill** [comment]
@jellyfin-bot rebase

**Review by nielsvanvelzen — APPROVED** [review]

**Review by ferferga — APPROVED** [review]

File: src/components/cardbuilder/cardBuilder.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ import ServerConnections from '../ServerConnections';
                         return 100 / 14.2857142857;
                     }
                     if (screenWidth >= 1200) {
-                        return 100 / 16.666666666666666666;
+                        return 100 / 16.66666666666666;
```
[the remark this answers is not in the document]
Comment (ferferga) [reply src/components/cardbuilder/cardBuilder.js:148]: Do we need that much precision though?

[the remark this answers is not in the document]
Comment (dmitrylyzo) [reply src/components/cardbuilder/cardBuilder.js:148]: In fact, this is just `6` (cards in a row), and divider goes from https://github.com/jellyfin/jellyfin-web/blob/f8c9158f812aab939a9f37548cd64a78de785844/src/components/cardbuilder/card.scss#L544

[the remark this answers is not in the document]
Comment (thornbill) [reply src/components/cardbuilder/cardBuilder.js]: I've updated this so that it matches the precision of the other instances in this file.

**Review by crobibero — APPROVED** [review]

**Pull request merged**